cmd.go

```diff
@@ -199,16 +199,28 @@ func search(v *vault.Vault) repl.ActionFunc {
 
 func clip(v *vault.Vault) repl.ActionFunc {
 	return func(args []string) (string, error) {
-		if len(args) != 1 {
-			return "", fmt.Errorf("clip requires 1 argument. See help for usage.")
+		if len(args) < 1 {
+			return "", fmt.Errorf("clip requires at least 1 argument. See help for usage.")
 		}
 
 		location := args[0]
 		cred, err := v.Get(location)
 		if err != nil {
 			return "", err
 		}
-		err = secureclip.Clip(cred.Password)
+
+		var toClip string
+		if len(args) > 1 {
+			meta := args[1]
+			if val, ok := cred.Meta[meta]; ok {
+				toClip = val
+			} else {
+				return "", fmt.Errorf("%v metadata tag not found.")
```

Comment:
missing the value for `%v` here (`fmt.Sprintf("%v metadata tag not found.", meta)`)